notes/login/models.py

```diff
@@ -1,3 +1,12 @@
 from django.db import models
 
 # Create your models here.
+
+from django.db import models
+
+
+class User(models.Model):
+    email = models.CharField(max_length=30)
+    password = models.CharField(max_length=30)
```

Comment:
let's increase max_length to 128 for password field as we're gonna store encrypted password which may be longer than 30 chars

Reply:
also let's ensure that email is unique
```suggestion
    password = models.CharField(max_length=30, unique=True))
```